npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser.rb

```diff
@@ -34,6 +34,15 @@ class FileParser < Dependabot::FileParsers::Base
         )?$
```

Comment:
Can you add a test for this? I am trying to picture what a null dependency looks like but I am not as a familiar with the ecosystem

Reply:
With "Null dependencies" I didn't mean some fancy package manager concept, just meant literally `"dependencies": null` 😄

I added a spec!

Reply:
Thanks for clarifying :)